thread-racing/README.adoc

```diff
@@ -60,13 +60,13 @@ The application will be running at the following URL http://localhost:8080/{arti
 
 To start a race press the `Insert Coin` button. The page displays the names of the threads as they join the race. It then tracks the progress of each thread through the Batch, EE Concurrency, JAX-RS, JMS, and JSON stages of the race. Finally, it displays the official race results and championship standings.
 
-// Testing with Arquillian
-include::../shared-doc/run-arquillian-integration-tests-with-server-distribution.adoc[leveloffset=+2]
+// Server Distribution Testing
+include::../shared-doc/run-integration-tests-with-server-distribution.adoc[leveloffset=+2]
 // Undeploy the Quickstart
 include::../shared-doc/undeploy-the-quickstart.adoc[leveloffset=+2]
 
 // Build and run sections for other environments/builds
 ifndef::ProductRelease,EAPXPRelease[]
 include::../shared-doc/build-and-run-the-quickstart-with-provisioned-server.adoc[leveloffset=+1]
 endif::[]
-include::../shared-doc/build-and-run-the-quickstart-with-openshift.adoc[leveloffset=+1]
+include::../shared-doc/build-and-run-the-quickstart-with-openshift.adoc[leveloffset=+1]
```

Comment:
@emmartins I tested the application with helm on OpenShift but I found very difficult to use this adoc file.
In particular [this line](https://github.com/wildfly/quickstart/blob/main/shared-doc/helm-deploy-project.adoc?plain=1#L10) explains how to install the chart: "helm install {artifactId} -f charts/helm.yaml {helmChartName} --wait --timeout=10m0s".
IMO it is not obvious that I have to replace {helmChartName} with wildfly/wildfly. WDYT?

Reply:
This is due to wfly to eap rebranding needs, it uses an attribute configured at /shared-docs/attributes.adoc , the best way to work with docs is to keep rebuilding it when doing changes (please use -Pdocs or -Drelease for the doc rebuild)

Reply:
I see! Thanks!